spec/support/login_helper.rb

```diff
@@ -0,0 +1,24 @@
+module LoginHelper
+  def login(admin)
+    click_link "Login"
+    fill_in :user_email, with: admin.email
+    fill_in :user_password, with: admin.password
+    click_button "Sign in"
+    expect(current_path).to eq('/')
+    expect(page).to have_content("Signed in successfully.")
+  end
+
+  def login(user)
+    click_link "Login"
+    fill_in :user_email, with: user.email
+    fill_in :user_password, with: user.password
+    click_button "Sign in"
+    expect(current_path).to eq('/')
+    expect(page).to have_content("Signed in successfully.")
+  end
+end
```

Comment:
Review the 2 login methods here and consider how to combine them into one method.